plugins/algoltek-usb/fu-algoltek-usb-device.c

```diff
@@ -479,8 +484,15 @@ fu_algoltek_usb_device_write_firmware(FuDevice *device,
 
 	fu_device_sleep(FU_DEVICE(self), 1000);
 
-	if (!fu_algoltek_usb_device_ers(self, error))
+	if (!fu_algoltek_usb_device_ers(self, 0x20, 31, error))
+		return FALSE;
+	if (!fu_algoltek_usb_device_ers(self, 0x20, 63, error))
```

Comment:
out of interest, what do 31 and 63 signify? Could they be defines in the .rs file, or at as defines at the top of the file?

Reply:
They are the identification address for 128k and 256k firmware respectively. Due to the hardware design mechanism in algoltek products, this information needs to be sent.
And I added them as defines in fu-algoltek-usb-common.h file.